.circleci/config.yml

```diff
@@ -0,0 +1,175 @@
+# CircleCI 2.0 configuration file
```

Comment:
You should copy this file from a more updated tool. We already do publish_local and push in a couple. Check lizard and gocyclo.